File: variables/14-Variables.php
<?php
//Solicitar al usuario una distancia en metros y 
//transformar a km., cm. y mm.
echo "Por favor, introduce la distancia en metros: ";
$metros = readline();
$kilometros = $metros / 1000;
$centimetros = $metros * 100;
$milimetros = $metros * 1000;
echo "La distancia de $metros metros equivale a:";
echo "- $kilometros kilómetros" . PHP_EOL;
echo "- $centimetros centímetros" . PHP_EOL;
echo "- $milimetros milímetros" . PHP_EOL;
?>

File: Net/14-variables.cs
double distanciaMetros, km, cm, mm;
Console.WriteLine("Ingrese la distancia en metros:");
distanciaMetros = double.Parse(Console.ReadLine());
km = distanciaMetros / 1000;
cm = distanciaMetros * 100;
mm = distanciaMetros * 1000;
Console.WriteLine("Distancia: " + km + " km, " + cm + " cm, " + mm + " mm");
